src/middleware/src/storage/index.js

```diff
@@ -1,22 +1,22 @@
 import { isDefined } from 'kinvey-js-sdk/dist/export';
-import KinveyStorage from 'kinvey-js-sdk/dist/request/src/middleware/src/storage';
-import IndexedDB from './src/indexeddb';
-import WebSQL from './src/websql';
-import { LocalStorage } from './src/webstorage';
+import { Storage as KinveyStorage } from 'kinvey-js-sdk/dist/request/src/middleware/src/storage';
+import { IndexedDBAdapter } from './indexeddb';
```

Comment:
Yes the files did move. Please see the path change designated like this

`src/middleware/src/storage/src/indexeddb.js → src/middleware/src/storage/indexeddb.js`